dcos/http.py

```diff
@@ -1,11 +1,18 @@
+import getpass
+import sys
+from urlparse import urlparse
```

Comment:
I don't think this is py2/3 compatible.  You have to use `six.moves.urllib.parse`.  see `mesos.py`